### ISSUE TITLE
AVT 1 - React Tab, has a DAP violation on the custom label

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3359]
## Environment
macOS Mojave version 10.14.5
Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React

## Detailed Description
1. Navigate to the Custom Label Tab
2. Run DAP and the following issue is present:
<img width="1137" alt="Screen Shot 2019-07-10 at 4 22 10 PM" src="https://user-images.githubusercontent.com/21676914/61061239-7c3e8b00-a3c1-11e9-89af-55918e512b00.png">
Note: This is only a violation on the custom label tab.

**Comment by jendowns** [comment]
Personally I don't see this violation in DAP myself, so I wonder if we are using different rulesets? (I'm using the Feb 2019 ruleset.) It should be perfectly valid to nest a `<span>` inside an `<a>`, yes? 🤔 

But either way -- I think this is an easy, storybook-only fix. PR incoming 👍

**Comment by dakahn** [comment]
reopening to investigate

**Comment by tw15egan** [comment]
@dakahn any more investigation needed, or can this be closed?

**Comment by abbeyhrt** [comment]
I'm not seeing a violation on the custom label anymore but there are DAP violations on the Tabs disabled state. @dakahn should we close this issue and open one for those violations?
